libraries/standard/mqtt/include/mqtt_lightweight.h

```diff
@@ -327,25 +334,38 @@ MQTTStatus_t MQTT_GetSubscribePacketSize( const MQTTSubscribeInfo_t * pSubscript
 /**
  * @brief Serialize an MQTT SUBSCRIBE packet in the given buffer.
  *
+ * #MQTT_GetSubscribePacketSize should be called with @p pSubscriptionList
+ * before invoking this function. The @p remainingLength was calculated from the
+ * parameters in @p pSubscriptionList using #MQTT_GetSubscribePacketSize.
+ * @p pSubscriptionList and @p remainingLength are valid only if
```

Comment:
pSubscriptionList and remainingLength are input params - what is meant by they are only valid if MQTT_GetSubscribePacketSize returned success?

Reply:
MQTT_GetSubscribePacketSize() also inputs pSubscriptionList and outputs remainingLength, so if MQTT_GetSubscribePacketSize doesn't return success, then these are invalid.